ports/stm/boards/stm32f411ce_blackpill/mpconfigboard.mk

```diff
@@ -4,7 +4,10 @@ USB_PRODUCT = "stm32f411ce blackpill"
 USB_MANUFACTURER = "WeAct"
 
 # SPI_FLASH_FILESYSTEM = 1
-# EXTERNAL_FLASH_DEVICES = xxxxxx #See supervisor/shared/external_flash/devices.h for options
+
+# See supervisor/shared/external_flash/devices.h for options
```

Comment:
[I know this was already merged, but: I think this comment is out of date, this info is in nvm.toml now]